src/boundaries.rs

```diff
@@ -213,17 +234,20 @@ pub fn build_boundary_parts<T: Borrow<osmpbfreader::OsmObj>>(
                         y: n.lat(),
                     })
                 };
-                let distance =
-                    p(poly_geom.first().unwrap()).haversine_distance(&p(poly_geom.last().unwrap()));
-                if distance < 10. {
-                    warn!(
-                        "boundary: relation/{} ({}): unclosed polygon, dist({:?}, {:?}) = {}",
-                        relation.id.0,
-                        relation.tags.get("name").map_or("", |s| &s),
-                        poly_geom.first().unwrap().id,
-                        poly_geom.last().unwrap().id,
-                        distance
-                    );
+
+                if added_nodes.len() > 1 {
+                    let distance = p(added_nodes.first().unwrap())
+                        .haversine_distance(&p(added_nodes.last().unwrap()));
+                    if distance < 10. {
```

Comment:
It may be better to use `10.` as a previously named constant? Something like `MAX_CYCLE_CLOSING_DISTANCE`?

*Maybe this should be left for another PR though?*

Reply:
done